keras_cv/datasets/pascal_voc/load.py

```diff
@@ -69,10 +73,13 @@ def load(
             For a list of supported formats, please  Refer
             [to the keras.io docs](https://keras.io/api/keras_cv/bounding_box/formats/)
             for more details on supported bounding box formats.
-        batch_size: how many instances to include in batches after loading
-        shuffle: whether or not to shuffle the dataset.  Defaults to True.
+        batch_size: (Optional) how many instances to include in batches after loading. If
+            not provided, no batching will occur.
+        shuffle: whether or not to shuffle the dataset.  Defaults to None, which indicates
```

Comment:
If that's the case, then we don't need a `shuffle` argument. Just `shuffle_buffer` being None would mean no shuffling, and being int would mean shuffling.